### ISSUE TITLE
Invalid line number is reported in line-continuation

### ISSUE THREAD

**Issue by miyakogi** [Kuniwak/vint#107]
Example

```vim
let g:v = {'a': 1}
let g:a = deepcopy(
      \ g:v["a"])
```

otuput

```
vint_check.vim:2:13: Prefer single quoted strings (see Google VimScript Style Guide (Strings))
```

Line 3 should be reported as an error but line 2 is reported.

- vint version 0.3.0

**Comment by Kuniwak** [comment]
Thanks for reporting the bug.

The cause of this bug is in [vim-vimlparser](https://github.com/ynkdir/vim-vimlparser).
I'm gonna report the bug to vim-vimlparser.

**Comment by miyakogi** [comment]
I see, thanks.

**Comment by Kuniwak** [comment]
Reported: https://github.com/ynkdir/vim-vimlparser/issues/16

**Comment by Kuniwak** [comment]
The bug will be fixed by the next release v0.3.1.
